src/core/plugins/samples/fn.js

```diff
@@ -233,6 +233,10 @@ export const sampleFromSchemaGeneric = (schema, config={}, exampleOverride = und
 
     // generate xml sample recursively for object case
     if(type === "object") {
+      // case literal example
+      if(typeof sample === "string") {
+        return sample
+      }
```

Comment:
Hi @mathis-m, thanks for looking into this!

I think this check for literal examples should also be added to `type: array` schemas - i.e. for any non-primitive schemas. What do you think?

Use case:
```yaml
type: array
xml:
  name: items
  wrapped: true
items:
  type: string
  xml:
    name: item
example: |-
  <items>
    <foo>foo<foo>
    <foo>bar<foo>
  </items>
```

Reply:
done